packages/in_app_purchase/test/fake_platform_views_controller.dart

```diff
@@ -0,0 +1,28 @@
+import 'dart:async';
+import 'package:flutter/services.dart';
+
+class FakePlatformViewsController {
```

Comment:
I'd expect a "FakePlatformViewsController" to be a fake implementation of the engine's [PlatformViewsController](https://github.com/flutter/engine/blob/master/shell/platform/android/io/flutter/plugin/platform/PlatformViewsController.java)